contracts/partial/bucket/bucket_methods.ligo

```diff
@@ -173,7 +185,10 @@ function default(
   var s                 : storage_t)
                         : return_t is
   block {
-    s.next_reward := s.next_reward + (Tezos.amount / 1mutez);
+    const baker_reward = Tezos.amount / 1mutez;
```

Comment:
I would name this one just `reward`, because having word `baker` confuses this variable with `baker_fund`